plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -198,6 +198,26 @@ export function AboutContent(props: AboutContentProps) {
           <Chip key={t} size="small" label={t} />
         ))}
       </AboutField>
+      <AboutField
+        label="Labels"
+        value="No Labels"
+        gridSizes={{ xs: 12, sm: 6, lg: 4 }}
+      >
+        {(Object.keys(entity?.metadata?.labels || {}) as (keyof string)[]).map(
```

Comment:
With `Object.entries` here you can avoid the additional lookup, and it should be fine to skip the cast too

Reply:
Hi,
Thanks @Rugvip for the suggestion. I have made the necessary changes.